### DISCUSSION TITLE
Complement to talk about Pop!_os on irc chat in real time

### DISCUSSION THREAD

**Discussion by vlcg-gif** [pop-os/pop#3233] [General]
To access our chat room in real time and exchange experiences with other users about pop, visit: https://web.libera.chat/#pop!_os

**Comment by jacobgkau** [comment]
The official chat is a Mattermost instance at https://chat.pop-os.org, but it's cool someone made a LiberaChat channel about it.

**Reply by vlcg-gif** [reply]
Ah, cool, I think I've seen something. I like IRC because I think it's less bureaucratic, there's no need to register. In addition to having other channels on different and imaginable topics.

**Reply by mmstick** [reply]
Account logins makes moderation easier as it prevents most spam. It also allows emails to be sent when someone responds to your message when you are not online. People who join an IRC chat cannot see or search past messages. Mattermost does have multiple channels, top level comments can be replied to directly in threads, and past comments can be searched.

**Comment by vlcg-gif** [comment]
But the option is there for anyone who wants to access it. https://hexchat.github.io/ if you want to enter through hexchat it's also worth it

**Reply by jackpot51** [reply]
Hexchat is no longer being maintained: https://hexchat.github.io/news/2.16.2.html

**Reply by vlcg-gif** [reply]
Hello, were you unable to download and access through Hexchat?

**Reply by leviport** [reply]
He's saying that you should recommend a different IRC client. One that is still being maintained.

**Reply by vlcg-gif** [reply]
Oh I see, Levi. Thanks!
In this article there are several that can serve as clients to connect to Irc.
https://libera.chat/guides/clients
